promotion/claim.go

```diff
@@ -84,6 +86,12 @@ func (service *Service) ClaimPromotionForWallet(
 	if promotion == nil {
 		return nil, errors.New("promotion did not exist")
 	}
+	if !promotion.Active {
```

Comment:
can you also cover the case where we've passed the 3 month claim window on the promotion?

Reply:
completed in: ea5f7a4